flink-core/src/main/java/org/apache/flink/configuration/NettyShuffleEnvironmentOptions.java

```diff
@@ -379,6 +380,28 @@ public class NettyShuffleEnvironmentOptions {
                                     + "tasks have occupied all the buffers and the downstream tasks are waiting for the exclusive buffers. The timeout breaks"
                                     + "the tie by failing the request of exclusive buffers and ask users to increase the number of total buffers.");
 
+    @Documentation.Section(Documentation.Sections.ALL_TASK_MANAGER_NETWORK)
+    @Experimental
+    @Internal
+    public static final ConfigOption<Boolean> NETWORK_HYBRID_SHUFFLE_ENABLE_TIERED_STORAGE =
+            ConfigOptions.key("taskmanager.network.hybrid-shuffle.enable-tiered-storage")
+                    .booleanType()
+                    .defaultValue(false)
+                    .withDescription(
+                            "The option is used to enable tiered storage architecture for hybrid shuffle mode.");
+
+    @Documentation.Section(Documentation.Sections.ALL_TASK_MANAGER_NETWORK)
+    @Experimental
+    @Internal
+    public static final ConfigOption<String> NETWORK_HYBRID_SHUFFLE_REMOTE_STORAGE_BASE_PATH =
+            key("taskmanager.network.hybrid-shuffle.remote.path")
+                    .stringType()
+                    .noDefaultValue()
+                    .withDescription(
+                            "The base home path of remote storage for remote tier. If the option is configured, "
+                                    + "Hybrid Shuffle will use the remote storage path as a supplement to the"
+                                    + "local disks. If not configured, the remote storage will not be used.");
+
```

Comment:
Is there any config option that only the old mode of hybrid shuffle uses but not the new one?